README.md

```diff
@@ -4,3 +4,57 @@
 ---
 
 This is the basic readme for this module. Include any usage or deployment instructions and links to other documentation here.
+
+
+## Contributing
+---
+This is a [KBase](https://kbase.us) module generated by the [KBase Software Development Kit (SDK)](https://github.com/kbase/kb_sdk).
+
+You will need to have the SDK installed to use this module if you want to make any changes to the spec or the API. However, other development work does not require the SDK and can be done locally.
+
+[Learn more about the SDK and how to use it](https://kbase.github.io/kb_sdk_docs/).
+
+## Set up
+
+The SetAPI repo has been developed and tested using Python 3.10. There are three sets of dependencies for the repo:
+
+* `requirements.txt` - packages required for runtime
+* `requirements-test.txt` - packages used in testing and generating coverage
+* `requirements-dev.txt` - includes a range of extra tools for formatting and linting code.
+
+The contents of the three files are mutually exclusive, so all three should be installed for the full development environment.
+
+Pip is not always completely reliable at pulling in package dependencies, so it is recommended that you install packages one at a time as follows:
+
+```bash
+$ cat requirements.txt | sed -e '/^\s*#.*$/d' -e '/^\s*$/d' | xargs -n 1 pip install
+```
+
+...substituting in the appropriate requirements file name.
```

Comment:
this is a C&P from the static narrative, which was recently modernised to remove the SDK from the docker image. I intend to do the same thing with this repo.